geograph/geograph.py

```diff
@@ -759,7 +841,7 @@ def get_metric(
     def get_class_metrics(
         self,
         names: Optional[Union[str, Sequence[str]]] = None,
-        classes: Optional[Union[str, Sequence[int], np.ndarray]] = None,
+        classes: Optional[Union[str, Sequence[Union[str, int]], np.ndarray]] = None,
```

Comment:
Haha 😄 since we use `Union[str, int]` quite a lot, we might want to think of defining a type shortcut: `Label=[Union[str,int]]` or so?

Reply:
Could be useful, but I think there aren't actually that many of these at the moment - maybe in future.